File: ios/ReactNativeModules/LoggerModule.m
//
//  LoggerModule.m
//  ReactNativeHybrid
//
//  Created by Oscar Cheung on 20/02/2023.
//

#import "React/RCTBridgeModule.h"

@interface RCT_EXTERN_MODULE(LoggerModule, NSObject)

  RCT_EXTERN_METHOD(log:(NSString *) message)

@end

File: ios/ReactNativeModules/LoggerModule.swift
//
//  LoggerModule.swift
//  ReactNativeHybrid
//
//  Created by Oscar Cheung on 20/02/2023.
//

@objc(LoggerModule)
class LoggerModule: NSObject {
  @objc
  static func requiresMainQueueSetup() -> Bool { return false }

  @objc
  func log(_ message: String) -> Void {
    print("LoggerModule:", message)
  }
}
